docker-compose.yml

```
@@ -43,7 +43,7 @@ services:
       - "443:443"
     volumes:
       - ./frontend/static_vol:/var/www/html
-      - ${CERTS}:/etc/nginx/ssl
+      - ${CERTS}:/etc/nginx/ssl:ro
```

ここにReadOnlyがつくと証明書がGeneratingできなくなるので
外していただけると🙏
```suggestion
      - ${CERTS}:/etc/nginx/ssl
```

テストしてみたらこういう挙動になりました。↓

まず、ローカルの${CERTS}にある証明書を削除する
→`docker-compose up --build -d`
→`- ${CERTS}:/etc/nginx/ssl`によってマウントされるが証明書はない
→ないので作る、しかしReadOnlyで書き込めず作れない
→ frontendコンテナが起動しない

---

ありがとうございます！
ro消しました

---

早速ありがとうございます！